crates/fluvio-protocol-derive/Cargo.toml

```diff
@@ -13,12 +13,8 @@ proc-macro = true
 doctest = false
 
 [dependencies]
+syn = { workspace = true, features = ["full"] }
 proc-macro2 = { workspace = true }
 quote = { workspace = true }
 tracing = { workspace = true }
-
-[dependencies.syn]
-version = "1.0.0"
-features = ["full"]
-
-
+anyhow = { workspace = true }
```

Comment:
We use `anyhow` only once in tests, and the usage is no different from `core::result::Result`, so we can drop it.